examples/src/main/scala/github/contributors/api/Api.scala

```diff
@@ -0,0 +1,39 @@
+package github.contributors.api
+
+import akka.http.interop.{HttpServer, ZIOSupport}
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import github.contributors.domain.ContributorService
+import de.heikoseeberger.akkahttpcirce.FailFastCirceSupport._
+import zio._
+import zio.config.ZConfig
+
+object Api {
+
+  private val url = "https://api.github.com/repos/zio/zio-redis/contributors"
```

Comment:
Well, this should be assembled. See the ticket spec. Hostname can be hardcoded, but make constants final and capitalized.